content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -891,6 +892,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   to running pods.
 - `EvenPodsSpread`: Enable pods to be scheduled evenly across topology domains. See
   [Pod Topology Spread Constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/).
+- `EventedPLEG`: Enable `EventedPLEG` for the kubelet. `EventedPLEG` also needs to be enabled in the CRI runtime
+   for this feature to work. If the CRI runtime does not implement `EventedPLEG` then kubelet will automatically switch to existing `GenericPLEG` even after enabling this feature.
```

Comment:
- What is an evented PLEG? Please at least explain the abbreviation.
- You can write `{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}` which renders as “container runtime” with a tooltip and a hyperlink to more info; no need to mention that the runtime is CRI compatible because Kubernetes doesn't support anything else.
- `GenericPLEG` isn't a feature gate name, so write “the legacy generic PLEG”.

Reply:
@sftim Thanks for your feedback. I made some changes based on it, please let me know what do you think. Thanks.